docs/content/topics/project-developers/secrets.md

```diff
@@ -9,239 +9,157 @@ aliases:
 
-Imagine a case where we need to pass a sensitive piece of information to one of the jobs
-in our `brigade.js`. For example, we might need to pass an authentication token to a job that
-must authenticate to a remote service.
+Imagine a case where we need to pass a sensitive piece of information to one of
+the jobs in a project's Brigade script. For example, we might need to pass an
+authentication token to a job that must authenticate to a remote service.
 
-We do this by storing the secret inside of the project definition.
+We first need to set the secret key and value on the project via `brig`. There
+are two methods to do this, both provided on the `brig project secret set`
+command:
 
-Project definitions are typically managed by `brig`. As you may recall from the [installation
-manual](../intro/install.md), a new project is created like via `brig project create`.
+ * Set a secret in-line:
 
-During the creation process, we are able to add secrets when prompted. If we'd forgotten to
-add a secret and/or additional secrets need to be added to this existing project, we can do so via
-`brig project create --replace -p <existing project name>`.
+  ```console
+  $ brig project secret set --project my-project --set foo=bar
+  ```
 
-Here we add a new secret with key `dbPassword` and value `supersecret` to our existing
-`brigadecore/empty-testbed` project:
+  * Setting a secret via a `secrets.yaml` file:
 
-```console
-$ brig project create --replace -p brigadecore/empty-testbed
-? Existing Project Name brigadecore/empty-testbed
-? Full repository name github.com/brigadecore/empty-testbed
-? Clone URL (https://github.com/your/repo.git) https://github.com/brigadecore/empty-testbed.git
-? Add secrets? Yes
-? 	Secret 1 dbPassword
-? 	Value supersecret
-? ===> Add another? No
-Auto-generated a Shared Secret: "9qQvuplBx39r04Wd9EmyxjGA"
-? Configure GitHub Access? No
-? Configure advanced options No
-Project ID: brigade-830c16d4aaf6f5490937ad719afd8490a5bcbef064d397411043ac
-```
+  ```console
+  $ echo "foo: bar" >> ./secrets.yaml
+  $ brig project secret set --project my-project --file ./secrets.yaml
+  ```
 
-## Accessing a Secret within `brigade.js`
+## Accessing a Secret within a Brigade script
 
-Within the `brigade.js` file, we can access any of the secrets defined on our project.
+Within a Brigade script (`brigade.js` or `brigade.ts` file), we can access any
+of the secrets defined on our project.
 
-```javascript
-// THIS IS NOT SAFE! IT LEAKS YOUR SECRET INTO THE LOGS.
-const {events, Job, Group} = require("brigadier")
-
-events.on("push", function(e, project) {
-  console.log("My DB password is " + project.secrets.dbPassword)
-})
-```
-
-Secrets can be selectively passed to jobs by setting environment variables.
+This first example demonstrates direct access of a secret defined on a project
+in the event handler code outside of any Job:
 
 ```javascript
 // THIS IS NOT SAFE! IT LEAKS YOUR SECRET INTO THE LOGS.
```

Comment:
Can we move this comment closer to the console.log command? I think that would be more clear about what, specifically, is unsafe.